src/main/java/org/grails/plugins/springsession/config/SpringSessionConfig.java

```diff
@@ -1,36 +1,24 @@
 package org.grails.plugins.springsession.config;
 
-import grails.core.GrailsApplication;
 import org.grails.plugins.springsession.converters.GrailsJdkSerializationRedisSerializer;
 import org.grails.plugins.springsession.web.http.HttpSessionSynchronizer;
 import org.springframework.boot.web.servlet.FilterRegistrationBean;
 import org.springframework.context.annotation.Bean;
 import org.springframework.core.Ordered;
 import org.springframework.data.redis.serializer.RedisSerializer;
-import org.springframework.data.redis.serializer.StringRedisSerializer;
 import org.springframework.session.Session;
 import org.springframework.session.data.redis.config.annotation.web.http.EnableRedisHttpSession;
 import org.springframework.session.web.http.SessionRepositoryFilter;
 import redis.clients.jedis.JedisPoolConfig;
 
 @EnableRedisHttpSession
 public class SpringSessionConfig {
-    private GrailsApplication grailsApplication;
-
-    public void setGrailsApplication(GrailsApplication grailsApplication) {
-        this.grailsApplication = grailsApplication;
-    }
 
     @Bean
-    public RedisSerializer jdkSerializationRedisSerializer() {
+    public RedisSerializer springSessionDefaultRedisSerializer() {
```

Comment:
@RXL-ViktorKhmelevsky as per your comment, now we don't need to jdkserializer spring session will use its own default serializer?

Reply:
it will look for `springSessionDefaultRedisSerializer` in the context